### PR TITLE
Spanish tutorials

### PR DESCRIPTION
**Pull request by nomadscientist**
This is the first human-translated Spanish tutorial, as part of a test/study of human-translated tutorials.
The tags/order of tutorials will need a bit of help, as the tag/subtopic wasn't working properly in my local render.

### CODE REVIEW

File: topics/introduction/metadata.yaml [hunk 1]
```diff
@@ -19,3 +19,6 @@ subtopics:
   - id: r
     title: "R"
     description: "These lessons focus on coding in R, using Galaxy analysis outputs as their basis."
+  - id: espanol
+    title: "Español"
+    description: "Estas prácticas son traducidas a español por ioinformáticas."
```
Comment (beatrizserrano) [inline topics/introduction/metadata.yaml:24]:
```suggestion
    description: "Estas prácticas son traducidas a español por bioinformáticas."
```

File: topics/introduction/tutorials/galaxy-intro-short-ES/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+> ![login](../../images/galaxy-login.png)
+>
+>   > ### {% icon comment %} Diferentes servidores Galaxy
+>   >  Esta es una imagen de Galaxy Australia, situado en [usegalaxy.org.au](https://usegalaxy.org.au/)
+>   >
+>   > El servidor de Galaxy que estas usando podría lucir un poco distinto y tener otra dirección web:
+>   > - El servidor principal de Galaxy es [usegalaxy.org](https://usegalaxy.org/)
+>   > - El servidor europeo de Galaxy es [usegalaxy.eu](https://usegalaxy.eu/)
+>   >
+>   > También puedes encontrar más servidores de Galaxy posibles en la parte superior de este tutorial en **Available on these Galaxies**
```
Comment (beatrizserrano) [inline topics/introduction/tutorials/galaxy-intro-short-ES/tutorial.md:65]:
```suggestion
>   > También puedes encontrar otros servidores de Galaxy en la parte superior de este tutorial en **Available on these Galaxies**
```

File: topics/introduction/tutorials/galaxy-intro-short-ES/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+> 1. Abre tu navegador favorito (usa Chrome, Safari o Firefox, no Internet Explorer!)
+> 2. Navega hasta tu instancia de Galaxy
+> 3. Inicia sesión o regístrate
+>
+> ![login](../../images/galaxy-login.png)
+>
+>   > ### {% icon comment %} Diferentes servidores Galaxy
+>   >  Esta es una imagen de Galaxy Australia, situado en [usegalaxy.org.au](https://usegalaxy.org.au/)
+>   >
+>   > El servidor de Galaxy que estas usando podría lucir un poco distinto y tener otra dirección web:
```
Comment (beatrizserrano) [inline topics/introduction/tutorials/galaxy-intro-short-ES/tutorial.md:61]:
```suggestion
>   > El servidor de Galaxy que estás usando podría lucir un poco distinto y tener otra dirección web:
```

File: topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga [hunk 4]
```diff
@@ -0,0 +1,161 @@
+{
+  "a_galaxy_workflow": "true",
```
Comment (hexylena) [inline topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga:2]: If nothing needs to be translated here, could probably symlink the folder.

**Comment by shiltemann** [comment]
Thanks @nomadscientist !

In order to keep the English and Spanish versions in sync, I would like to start a github team of Spanish speakers who are willing to help out with updating the Spanish version whenever somebody suggests an update to the original. So please let me know who I should be adding to this team.

@beatrizserrano @gallardoalba and any other Spanish speaking GTN members, would you be willing to be part of this team? i.e. get pinged when translations are needed and help out if you can?

**Comment by gallardoalba** [comment]
> Thanks @nomadscientist !
> 
> In order to keep the English and Spanish versions in sync, I would like to start a github team of Spanish speakers who are willing to help out with updating the Spanish version whenever somebody suggests an update to the original. So please let me know who I should be adding to this team.
> 
> @beatrizserrano @gallardoalba and any other Spanish speaking GTN members, would you be willing to be part of this team? i.e. get pinged when translations are needed and help out if you can?

Yes, count on me.

**Comment by beatrizserrano** [comment]
sure, I can help! :)

**Comment by shiltemann** [comment]
Thanks! @martenson, could you create a github team named `training-spanish` and add the members?

TODO for myself:
- [ ] update GTN framework to support manual translations in the language dropdown menu

**Comment by nomadscientist** [comment]
And vice-versa as well - Pati has found a few typos already that have
helped clean up the English version, so it definitely works both ways! And
I can definitely see a future wherein a tutorial is submitted FIRST in
Spanish, and translated to English (and that one I can help with)

File: topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga (hunk 4 above)
Comment (nomadscientist) [reply topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga:2]: Symlink the folder? What does this mean?

Comment (nomadscientist) [reply topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga:2]: OH as in nab the workflow link from the original? In which case... yes!

Comment (hexylena) [reply topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga:2]: Oh, it's a symbolic link, a pointer to the folder that still behaves like a folder. I can do it tomorrow! :)

Comment (hexylena) [reply topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga]: Converted workflows folder to symlink. https://github.com/galaxyproject/training-material/pull/2534/files#diff-57f27e497983460cff1807a3e1b5ee8cda56494146cceaf8ead3b985737040a3 That way we can ensure the WF stays in sync with the english version.

Comment (nomadscientist) [reply topics/introduction/tutorials/galaxy-intro-short-ES/workflows/Galaxy-Workflow-galaxy-intro-short.ga]: Ace!

**Comment by shiltemann** [comment]
I restructured it a bit @nomadscientist, so now you get both curated and automatic translation in the dropdown:

image

and a tag on the tutorial, what do you think?

**Comment by nomadscientist** [comment]
> > I restructured it a bit @nomadscientist, so now you get both curated and automatic translation in the dropdown:
> > image
> > and a tag on the tutorial, what do you think?
> 
> I think that's bloomin' brilliant!

Actually - will we be able to see how often somebody clicks on either the tag or the curated translation?

**Comment by shiltemann** [comment]
@nomadscientist yes, on every tutorial there is now a "page metrics" button (see "extras" menu at top), this will show the number of page views over time :)

**Comment by shiltemann** [comment]
@nomadscientist ok, added the curated translation option to the Extras menu as well:

image

(and same for going back to the english version from spanish)

**Comment by shiltemann** [comment]
also, shouldn't you be listed as a contributor for the Spanish version of the tutorial as well?

**Comment by shiltemann** [comment]
@nomadscientist I've also added a line to the CODEOWNERS file so that you will get notified of any changes proposed to the tutorials. Feel free to add your other collaborators there as well, if they are happy to receive the notifications as well

**Review by shiltemann — DISMISSED** [review]

**Comment by davelopez** [comment]
> would you be willing to be part of this team? i.e. get pinged when translations are needed and help out if you can?

Please feel free to ping me as well if I can help with some Spanish translations.

**Comment by nomadscientist** [comment]
Well I didn't make or translate that one, so felt a bit wrong to be a
contributor. But pinged with changes, yes absolutely

**Comment by nomadscientist** [comment]
VERY cool on the curated button. VERY cool.

**Comment by shiltemann** [comment]
thanks @davelopez!!

**Review by shiltemann — APPROVED** [review]

**Comment by nomadscientist** [comment]
Wahooo!!!!!  I saw the codeowners file - that means we all get notified no matter what gets changed or flagged, right? ie
topics/introduction/tutorials/galaxy-intro-short/     @nomadscientist @beatrizserrano @gallardoalba @davelopez

**Comment by nomadscientist** [comment]
"nomadscientist requested a review from galaxyproject/training-transcriptomics as a code owner 29 minutes ago" <--- how did I manage this? Fascinating.

**Comment by shiltemann** [comment]
> Wahooo!!!!! I saw the codeowners file - that means we all get notified no matter what gets changed or flagged, whether English or Spanish or whatever, right?

yes exactly :)

**Review by hexylena — APPROVED** [review]

**Pull request merged**